lib/mobility-core/src/Kernel/External/Notification/PayTM/Types.hs

```diff
@@ -37,7 +39,10 @@ data PayTMConfig = PayTMConfig
   }
   deriving (Show, Eq, Generic, FromJSON, ToJSON)
 
-data NotificationReceiverType = CUSTOMERID deriving (Generic, Show, FromJSON, ToJSON)
+data NotificationReceiverType = CUSTOMERID deriving (Generic, Show, FromJSON)
```

Comment:
Your `FromJSON` and `ToJSON` instances is not corresponding each other.
I mean `fromJSON . toJSON $ CUSTOMERID` should return `CUSTOMERID` but will fail
So, `FromJSON` instance also should be custom